packages/affiliates/gas-rebate/rebates/Rebate_45.json

```diff
@@ -0,0 +1,140 @@
+{
+  "votingContractAddress": "0x004395edb43EFca9885CEdad51EC9fAf93Bd34ac",
+  "rebate": 45,
+  "fromBlock": 19993250,
+  "toBlock": 20200783,
```

Comment:
This is Jun-30-2024 12:00:11 AM +UTC, but next month would start ~Jul-1 12:00 AM, so there would be 24h gap

Reply:
This might have something to do with locale settings. I recall similar issues last month and resolved by changing my machine timezone to UTC when running the script

Reply:
Nice catch @Reinis-FRP ! I've update it by running with the latest block of june like this:
`OVERRIDE_TO_BLOCK=20207948 yarn hardhat run ./gas-rebate/VoterGasRebateV2.ts --network mainnet`